adoc/chapters/glossary.adoc

```diff
@@ -284,6 +284,15 @@ For the full description please refer to <<subsec:buffers>>.
   * [code]#sycl::group# : The <<group>> class that contains the
     <<work-group-id>> and the member functions on a <<work-group>>.
 
+[[information-descriptor]]information descriptor::
+    A class encapsulating an information query and its return type.
+    For example, an information descriptor called
+    [code]#sycl::info::class::name# would describe a query for the name of the
+    entity represented by that class, and would define a return type of
+    [code]#std::string#.
```

Comment:
Why is the return type `std::string`?  Just because a "name" would normally be a string?

Reply:
Right, exactly.

I've tried to reword this glossary entry in b70832fed89922457c751e78a1e5c2c62b1dd619 to avoid this issue: instead of saying that it "would" return a `std::string` just because it's called "name", I've flipped it to say that if a descriptor was called "name" and had a return type defined to be `std::string`, the information query should be expected to return the entity's name as a string.